drf_braces/utils.py

```diff
@@ -130,6 +141,23 @@ def get_class_name_with_new_suffix(klass, existing_suffix, new_suffix):
 
 
 def get_attr_from_base_classes(bases, attrs, attr, **kwargs):
+    """
+    The base class attributes are retrieved if they are not already
+    present on the object.
+
+    Args:
+        bases (tuple, list): The base classes for a class
+        attrs (dict): The attributes of the class
+        attr (str): Specific attribute being looked for
+
+    Returns:
+        attribute value (str) or a default which is expected.
```

Comment:
`attribute value as found in base classes or a default when attribute is not found and default is provided.`